src/copilot/inspect/InspectionCopilot.ts

```diff
@@ -220,10 +220,8 @@ export default class InspectionCopilot extends Copilot {
         const adjustedRange = new Range(new Position(range.start.line, 0), new Position(range.end.line, document.lineAt(range.end.line).text.length));
         const content: string = document.getText(adjustedRange);
         const startLine = range.start.line;
-        const javaVersion = await getProjectJavaVersion(document);
-        const inspections = await this.inspectCode(content, {
-            javaVersion
-        });
+        const context = await this.collectProjectContext(document);
+        const inspections = await this.inspectCode(content, context);
```

Comment:
First glance, we just thought `content` and `context` are the same word. Maybe rename `context` -> `javaContext` ?

Reply:
I will rename it to `projectContext` because it may contains information not related to java.